black/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6-alpine3.7
+
+RUN addgroup -S black && adduser -D -S black black && \
+    apk add --no-cache bash && \
+    pip install black==18.4a2 && \
+    mkdir /src
+
+LABEL name=black version=dev \
```

Comment:
Version seems to be 18.4a2 here for me, but we nowadays omit both name and version for new images